### DISCUSSION TITLE
InvalidConsistencyLevel

### DISCUSSION THREAD

**Discussion by miche2020** [milvus-io/milvus#15884] [Show and tell]
I execute tutorial code and have some issues with queries parmas  consistency_level="strong"
![image](https://user-images.githubusercontent.com/66796995/156888889-71de3c37-6ab2-4e87-8491-c5a912bab1a6.png)
![image](https://user-images.githubusercontent.com/66796995/156888903-0be0ca71-c808-4377-9cb6-7ee57362dd92.png)

**Comment by xiaofan-luan** [comment]
@DragonDriver any ideas?

**Reply by miche2020** [reply]
> @DragonDriver any ideas?

oh seems error is 'Strong' not strong

**Comment by congqixia** [comment]
Hi @miche2020 , the consistency_level shall be "Strong" with capital S. Thanks for pointing out the mistake in the docs!

**Reply by xiaofan-luan** [reply]
We should be case sensitive?
